src/EVault/modules/Governance.sol

```diff
@@ -35,9 +35,20 @@ abstract contract GovernanceModule is IGovernance, Base, BalanceUtils, BorrowUti
     event GovSetGovernorAdmin(address indexed newGovernorAdmin);
     event GovSetFeeReceiver(address indexed newFeeReceiver);
     event GovSetLTV(
-        address indexed collateral, uint48 targetTimestamp, uint16 targetLTV, uint32 rampDuration, uint16 originalLTV
+        address indexed collateral,
+        uint16 borrowLTV,
+        uint16 liquidationLTV,
+        uint16 initialLiquidationLTV,
+        uint48 targetTimestamp,
+        uint32 rampDuration
     );
+    /// @notice Set a maximum liquidation discount
+    /// @param newLiquidationDiscount The new maximum liquidation discount in 1e4 scale
```

Comment:
Just out of curiosity, we only these events have the natspec? Is this an error, or are you planning to support all the events with natspec?

Reply:
The other events are commented in other pending branches. When merged, all of them will have natspec